### ISSUE TITLE
Better exception handling of org.reflections.vfs.DefaultUrlTypes.bundle.createDir()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#171]
```
The method org.reflections.vfs.DefaultUrlTypes.bundle.createDir() handles all 
possible problems with returning null.
Possible problems are:
- org.eclipse.equinox.common not in class path -> ClassNotFoundException
- org.eclipse.equinox.common not started -> InvocationTargetException with 
NullPointerException as target (source of NullPointerException is 
org.eclipse.core.internal.runtime.Activator.java returning null in getContext()
- resolve() itself returns null (for any reason)

Please distinguish between these cases so it is easier to understand, why 
createDir() returns null.

Thanks
```

Original issue reported on code.google.com by `aholt...@googlemail.com` on 11 Apr 2014 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
DefaultUrlTypes.bundle.createDir() _doesn't_ eat the exception and return null.
Vfs.fromURL would log.error in case of failure, and silently try the next 
VfsType - in that case look at the logs.
```

Original comment by `ronm...@gmail.com` on 5 Jun 2014 at 6:50
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
It got fixed in 0.9.9.RC2 (I was using RC1 when writing this issue)
Thanks a lot.
```

Original comment by `aholt...@googlemail.com` on 19 Jan 2015 at 4:38
